packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -265,6 +265,12 @@ export function isTaggedTelemetryPropertyValue(x: any): x is ITaggedTelemetryPro
     return (typeof (x?.value) !== "object" && typeof (x?.tag) === "string");
 }
 
+export function isSimpleArray(x: any): boolean {
```

Comment:
Some notes on this API:
1. It needs some documentation RE what "simple" means
2. Looking at implementation and usage, I assume you want to say you really want "is it jsonable", and incorporate some of the logic of getValidTelemetryProps into here in this format:

```ts
export function isJSONable(x: any): boolean {
   switch (typeof val) {
      case "string":
      case "number":
      case "boolean":
      case "undefined":
          return true;
      case default:
          if (!Array.isArray(x)) {
            return false;
          }
          return x.every((val) => isJSONable(val));
   }
}
```

Reply:
I also wonder if we can allow objects that way. This might be helpful for cases where we started to use `details` property that is JSONable object - it would be great if this shared code did strigification, and not sprinkle it across all the code.

Reply:
I opened a task about this, thanks for bringing it up Vlad:  https://dev.azure.com/fluidframework/internal/_workitems/edit/1104